pkg/tasks/task_action_builder.go

```diff
@@ -30,7 +30,8 @@ func (a *genericTaskActionBuilder) OnFunc(condFunc func(*Context) *actionResult)
 	return a
 }
 
-// OnFileChange specifies that the action will run when a file changes or does not exist.
+// OnFileChange specifies that the action will run when a file changes.
+// The action will NOT run if the file does not exist.
 // The action will NOT fail if the file is not created.
```

Comment:
In which context a file would be created?

Reply:
The task could create the file, or not, the condition is oblivious to that.

There is a discrepancy between the API and the current implementation.
- The `OnFileChange` suggests that it's only a **trigger**
- The implementation is adding a **validation** on top of the trigger

It was making sense when there was a single `met?` logic that could be twisted as needed.

If we see a need for a validation step, then we could implement it separately:
```go
builder := actionBuilder("", func(ctx *Context) error { return nil })
builder.OnFileChange("myfile")
builder.Validate(func(ctx *Context) error { return nil })
```